<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="type">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Goldak</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Elipsoid</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="type_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Heat Source Type:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_ok">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>330</y>
      <width>95</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>441</width>
      <height>241</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="home">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>321</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>Please select a heat source type...</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="goldak">
     <widget class="QTextEdit" name="goldak_a_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>104</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="goldak_cr_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>104</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="goldak_cr_label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>74</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>cr</string>
      </property>
     </widget>
     <widget class="QLabel" name="goldak_b_label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>74</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>b</string>
      </property>
     </widget>
     <widget class="QLabel" name="goldak_measurement_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Length (mm):</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="goldak_cf_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>104</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="goldak_image_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>0</y>
        <width>251</width>
        <height>201</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Goldak.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="goldak_a_label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>74</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>a</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="goldak_b_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>104</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="goldak_cf_label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>200</y>
        <width>74</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>cf</string>
      </property>
     </widget>
     <widget class="QLabel" name="z_direction_1">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>210</y>
        <width>201</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Welding direction - z axis</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ellipsoid">
     <widget class="QTextEdit" name="ellipsoid_a">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>104</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="ellipsoid_image">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>0</y>
        <width>251</width>
        <height>201</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Ellipsoid.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="ellipsoid_c_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>74</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>c</string>
      </property>
     </widget>
     <widget class="QLabel" name="ellipsoid_measurement">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Radius (mm):</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="ellipsoid_c">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>104</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="ellipsoid_a_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>74</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>a</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="ellipsoid_b">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>104</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="ellipsoid_b_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>74</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>b</string>
      </property>
     </widget>
     <widget class="QLabel" name="z_direction_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>210</y>
        <width>201</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Welding direction - z axis</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>494</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
